### PR TITLE
Update manifests for expand support

### LINKED ISSUE #86: Add support for volume resizing.
In preparation for volume resizing going beta, we should add support for it in hostpath driver and make sure the tests run with it.

### PR DESCRIPTION
**Pull request by bertinatto**
/kind feature

**What this PR does / why we need it**:

This PR updates the manifest files and deploy script in order to have volume expansion support.

This is a continuation of #87.

Fixes #86 

```release-note
Added volume expansion support.
```

### CODE REVIEW

**Comment by gnufied** [comment]
Can we double check the script:

```
error: unable to read URL "https://raw.githubusercontent.com/kubernetes-csi/external-resizer//deploy/kubernetes/rbac.yaml", server reported 404 Not Found, status code=404
```

**Comment by bertinatto** [comment]
CC @gnufied

File: deploy/util/deploy-hostpath.sh [hunk 1]
```diff
@@ -147,7 +149,7 @@ done
 # Wait until all pods are running. We have to make some assumptions
 # about the deployment here, otherwise we wouldn't know what to wait
 # for: the expectation is that we run attacher, provisioner,
-# snapshotter, socat and hostpath plugin in the default namespace.
+# snapshotter, resizer, socat and hostpath plugin in the default namespace.
 cnt=0
 while [ $(kubectl get pods 2>/dev/null | grep '^csi-hostpath.* Running ' | wc -l) -lt 5 ] || ! kubectl describe volumesnapshotclasses.snapshot.storage.k8s.io 2>/dev/null >/dev/null; do
```
Comment (pohly) [inline deploy/util/deploy-hostpath.sh]: I think this passed testing accidentally: it adds one additional pod (external-resizer), but then doesn't bump up the number of pods that the script waits for. This works if the script sees five running pods, but if only sees four and then six, the script waits until it times out.

That would explain https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-csi-canary-on-kubernetes-1-14/1191339260892942337 which doesn't have any obvious pod failures.

**Pull request merged**